scripts/system_performance/system_performance_queries.py

```diff
@@ -0,0 +1,156 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+
+"""
+The purpose of this script is for performing MySQL queries to monitor system state
+performance and health.
+"""
+from __future__ import print_function
+import itertools
+import logging
+from utils.clients.database_client import DatabaseClient
+
+
+class DatabaseMonitorChecks(object):
+    """Class to query system performance"""
+    table = "reduction_viewer_reductionrun"
```

Comment:
would this better as `self.table`? or is there a reason to access it this way rather than member var